docs/source/guide/guide-executors.rst

```diff
@@ -203,6 +203,84 @@ You can also include both noise models and finite sampling in your executor.
     assert noisy_output < 1.0
     assert noisy_output > 0.5
 
+.. _pyquil_executors:
+
+PyQuil Executors
+================
+
+This section contains executors for use with `pyQuil <https://github.com/rigetti/pyquil>`_.
+
+PyQuil: Quantum Cloud Services
+------------------------------
+
+This executor can be used to run on `Quantum Cloud Services <https://arxiv.org/abs/2001.04449>`_
+(QCS), the hardware platform provided by Rigetti Computing. Requires a QCS account and
+reservation on a quantum processor (QPU).
+
+In addition, ``mitiq_pyquil/executors.py`` has a function ``generate_qcs_executor`` for
+easily generating a QCS executor of this form from a template.
+
+Note that you will have to replace the string in ``get_qc`` with the name of an actual
+Rigetti QPU, and will need to have a QCS account and reservation, in order to run on
+real quantum hardware.
+
+.. testcode::
+
+    from pyquil import Program, get_qc
+    from pyquil.gates import MEASURE, RESET, X
+
+    from mitiq.mitiq_pyquil.compiler import basic_compile
+    from mitiq.mitiq_pyquil.executor import ground_state_expectation
+
+    # replace with qpu = get_qc("Aspen-8") to run on the Aspen-8 QPU
+    qpu = get_qc("2q-pyqvm")
```

Comment:
Also `get_qc("Aspen-8", as_qvm=True)` would work I think? Either is fine.

Reply:
Yes, but then the docs need a QVM server running, which is annoying. This version is python-only.